docs/modules/ROOT/pages/building.adoc

```diff
@@ -107,11 +107,131 @@ Contrary to earlier versions, version 1.7 and later are packaged as a `pkg` inst
 Do not call `make package` at any time when compiling for OS X, as this will build a disk image, which will not work correctly.
 ====
 
-[[windows-development-build]]
-== Windows Development Build
+[[windows-development-build-craft]]
```

Comment:
You don't need to add a custom anchor, as [they're generated automatically](https://asciidoctor.org/docs/user-manual/#auto-generated-ids).

Reply:
I was just following the pattern from the rest of the file :man_shrugging:

Reply:
Fair call. It's a new best practice that I'm doing my best to phase in.